roc/log_test.go

```diff
@@ -140,6 +141,47 @@ func TestLog_Func(t *testing.T) {
 	}
 }
 
+func TestLog_LogWrite(t *testing.T) {
+	SetLogLevel(LogDebug)
+	defer SetLogLevel(defaultLogLevel)
+
+	ch := make(chan LogMessage, 1)
+
+	SetLoggerFunc(func(msg LogMessage) {
+		if msg.Level == LogDebug {
+			select {
+			case ch <- msg:
+			default:
+			}
+		}
+	})
+	defer SetLoggerFunc(nil)
+
+	ctx, err := OpenContext(ContextConfig{})
+	require.NoError(t, err)
+	ctx.Close()
+
+	select {
+	case msg := <-ch:
+		assert.Equal(t, LogDebug, msg.Level, "Expected log level to be debug")
+		assert.NotEmpty(t, msg.Module)
+		assert.Equal(t, "roc_go", msg.Module)
+		assert.NotEmpty(t, msg.File)
+		assert.Equal(t, "log.go", msg.File[len(msg.File)-6:])
+		assert.NotEmpty(t, msg.Line)
+		assert.Equal(t, 192, msg.Line)
+		assert.NotEmpty(t, msg.Pid)
+		assert.Equal(t, uint64(os.Getpid()), msg.Pid)
+		assert.NotEmpty(t, msg.Tid)
+		assert.NotEmpty(t, msg.Time)
+		assert.Equal(t, time.Now().Truncate(time.Minute), msg.Time.Truncate(time.Minute))
+		assert.NotEmpty(t, msg.Text)
+		assert.Equal(t, "entering OpenContext(): config=%+v%!(EXTRA roc.ContextConfig={0 0})", msg.Text)
```

Comment:
* NotEmpty is redundant
* I think equality check is too strict here, we can just do some sanity check, e.g. ensure that text contains "OpenContext" or maybe "entering OpenContext"